src/Caliburn.Micro.Platform/Platforms/net46-netcore/Bootstrapper.cs

```diff
@@ -83,7 +83,19 @@ public void Initialize()
         protected virtual void StartDesignTime()
         {
             AssemblySource.Instance.Clear();
-            AssemblySource.Instance.AddRange(SelectAssemblies());
+			
+			foreach (var assembly in SelectAssemblies())
```

Comment:
Looks like the formatting is off here.

Reply:
Looks good when looking at the file alone. I was not able to see in the file, only in the diff. I look for this tomorrow. As I wrote in the other issue I was not able to build/open using Visual Studio to check the solution. So pre-configured  settings from you may not applied here.

Reply:
Looks good here https://github.com/beachwalker/Caliburn.Micro/blob/e07ea940ceb9d8b034333678357721ba48fc1315/src/Caliburn.Micro.Platform/Platforms/net46-netcore/Bootstrapper.cs#L87